vhdbuilder/scripts/linux/tool_installs.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-
-{{/* BCC/BPF-related error codes */}}
+# shellcheck disable=SC1072,SC1073,SC1127,SC1072
```

Comment:
dupe: SC1072

Reply:
actually aren't all three of these ignored in `.pipelines/scripts/verify_shell.sh` already?

Reply:
removing